### ISSUE TITLE
Feature: predictions

### ISSUE THREAD

**Issue by DC7IA** [la5nta/pat#217]
Not sure if this is something that is really hard to do.

Predictions are currently what I wish most for in pat.

Something like with Airmail, where it tells you the probability for a gateway and a frequency band.

After looking at my Airmail Virtual Machine, I figured out I could HB9AW on 80 m and just made my first PACTOR 3 connection with pat. :D 

(Previously, I had only used ARDOP.)

Anyone aware of any projects that could be used for predictions? ICEPAC ­- which gets installed with Airmail - seems to be closed source, unfortunately.

**Comment by DC7IA** [comment]
Just found this: https://github.com/jawatson/voacapl

I wonder if that could be of any use.

**Comment by DC7IA** [comment]
Predictions would make #136 (automatic connect mode) much easier, since we would know which station to connect to...rather than simply trying dozens of gateways until one can be reached.

**Comment by martinhpedersen** [comment]
I agree. This would be a very nice addition to the rmslist feature 😃

**Comment by DC7IA** [comment]
This might be interesting: https://github.com/jawatson/voacapl
